drivers/hmis/app/models/hmis/ce/configuration.rb

```diff
@@ -29,7 +29,8 @@ def days_to_retain_orphan_candidate_pools
     def values
       @values ||= AppConfigProperty.
         where(key: PROPERTIES.map { |attr| key_for(attr) }).
-        index_by(&:key)
```

Comment:
I think this is a bugfix; previously the `values` hash was mapping `key => AppConfigProperty`, now it maps `key => value`. A row in the DB `app_config_properties` table like 
`(id) | hmis_ce/enabled | false | ...`
should cause `enabled?` to return `false`, not `true`.

Reply:
Looks like a bug, thanks for fixing!